kolga/settings.py

```diff
@@ -90,6 +90,7 @@
     # ================================================
     "K8S_ADDITIONAL_HOSTNAMES": [env.list_none, []],
     "K8S_CLUSTER_ISSUER": [env.str, ""],
+    "K8S_INGRESS_ANNOTATIONS": [env.str, ""],
```

Comment:
Here you can use `env.list_none` which will turn the string into a list for you. That would then look like:
`"K8S_INGRESS_ANNOTATIONS": [env.list_none, []],`

That will also take care of the splitting at least for you, it will not do the key/value splitting.

Reply:
Thank you, I saw that but for some reason made self. Probably thought to use some other separator than comma, as separator can't be in the annotation value.